Cartfile.private

```diff
@@ -1,2 +1,2 @@
 binary "https://www.mapbox.com/ios-sdk/Mapbox-iOS-SDK.json" ~> 4.0
-github "AliSoftware/OHHTTPStubs" ~> 6.0
+github "frederoni/OHHTTPStubs" "563f48d3fab84ef04639649c770b00f4fa502cca"
```

Comment:
MapboxDirections.swift [depends on](https://github.com/mapbox/MapboxDirections.swift/blob/e72cb3b704dd3d357fc08e107a937b41bdd12d7c/Cartfile.private#L2) a different fork for AliSoftware/OHHTTPStubs#286; we should be consistent one way or the other.

Reply:
I'll migrate to this fork in MapboxDirections.